src/getInlineCompletionItems.ts

```diff
@@ -0,0 +1,54 @@
+import * as vscode from "vscode";
+import TabnineInlineCompletionItem from "./inlineSuggestions/tabnineInlineCompletionItem";
+import runCompletion from "./runCompletion";
+import getAutoImportCommand from "./getAutoImportCommand";
+import { SuggestionTrigger } from "./globals/consts";
+import { AutocompleteResult, ResultEntry } from "./binary/requests/requests";
+import { isMultiline } from "./utils/utils";
+
+export const INLINE_REQUEST_TIMEOUT = 3000;
+
+export default async function getInlineCompletionItems(
```

Comment:
mostly refactoring, this function was extracted from [provideInlineCompletionItems.ts](https://github.com/codota/tabnine-vscode/pull/993/files#diff-88c55a307a2aafd4bd44fa02d2ceb1a8fffa4246cde36e7cf4981e2df04b8e75)